wordpress-mobile/WordPress-FluxC-Android
# Localize account creation

Addresses the account related part of #609 by setting the `locale` body param of the ~`/users/new` and~ `auth/send-signup-email`.

To test:
1. Put the device on the French language/locale
2. Run the example app and head over to the "Signed out actions" section
3. Tap on "Send magic link sign e-mail" and input an email you have access to but *not* already registered with WPCOM and hit "Next"
4. Access that email account and verify that a new email has arrived and it's in French
